profile_bluesky/startup/instrument/devices/phaseplates.py

```diff
@@ -23,15 +66,76 @@ class PRDevice(Device):
     th = FormattedComponent(EpicsMotor,'{self.prefix}:{_motorsDict[th]}',
                             labels=('motor','phase retarders'))
 
+    _tracking = Component(Signal,value=False)
+    d_spacing = Component(Signal,value=0)
+    # TODO: Should d_spacing be an SignalRO to prevent users from changing
```

Comment:
It's a good idea _as long as temperature is constant_.  Once temperature varies, you might want to reconsider.  You _could_ raise a warning or even set a suspender to check for changes.